detector/wordpress.go

```diff
@@ -35,20 +36,37 @@ type WpCveInfos struct {
 
 // WpCveInfo is for wpscan json
 type WpCveInfo struct {
-	ID         string     `json:"id"`
-	Title      string     `json:"title"`
-	CreatedAt  time.Time  `json:"created_at"`
-	UpdatedAt  time.Time  `json:"updated_at"`
-	VulnType   string     `json:"vuln_type"`
-	References References `json:"references"`
-	FixedIn    string     `json:"fixed_in"`
+	ID            string     `json:"id"`
+	Title         string     `json:"title"`
+	CreatedAt     time.Time  `json:"created_at"`
+	UpdatedAt     time.Time  `json:"updated_at"`
+	PublishedDate time.Time  `json:"published_date"`
+	Description   string     `json:"description"` // Enterprise only
+	Poc           string     `json:"poc"`         // Enterprise only
+	VulnType      string     `json:"vuln_type"`
+	References    References `json:"references"`
+	Cvss          Cvss       `json:"cvss"` // Enterprise only
+	Verified      bool       `json:"verified"`
+	FixedIn       string     `json:"fixed_in"`
+	IntroducedIn  string     `json:"introduced_in"`
+	Closed        Closed     `json:"closed"`
 }
 
 // References is for wpscan json
 type References struct {
-	URL     []string `json:"url"`
-	Cve     []string `json:"cve"`
-	Secunia []string `json:"secunia"`
+	URL []string `json:"url"`
+	Cve []string `json:"cve"`
```

Comment:
It seems that reference also has a key called `youtube`.

https://www.youtube.com/watch?v=HaW15aMzBUM
```json
{
    "id": "4cd46653-4470-40ff-8aac-318bee2f998d",
    "title": "WordPress 3.7 to 5.7.1 - Object Injection in PHPMailer",
    "created_at": "2021-05-13T07:18:06.000Z",
    "updated_at": "2021-05-15T07:01:02.000Z",
    "published_date": "2021-05-13T00:00:00.000Z",
    "vuln_type": "OBJECT INJECTION",
    "references": {
        "url": [
            "https:\/\/github.com\/WordPress\/WordPress\/commit\/267061c9595fedd321582d14c21ec9e7da2dcf62",
            "https:\/\/wordpress.org\/news\/2021\/05\/wordpress-5-7-2-security-release\/",
            "https:\/\/github.com\/PHPMailer\/PHPMailer\/commit\/e2e07a355ee8ff36aba21d0242c5950c56e4c6f9",
            "https:\/\/www.wordfence.com\/blog\/2021\/05\/wordpress-5-7-2-security-release-what-you-need-to-know\/"
        ],
        "cve": [
            "2020-36326",
            "2018-19296"
        ],
        "youtube": [
            "HaW15aMzBUM"
        ]
    },
    "verified": true,
    "poc": null,
    "description": null,
    "cvss": null,
    "fixed_in": "4.9.18",
    "introduced_in": null
},
```

```suggestion
	URL []string `json:"url"`
	Cve []string `json:"cve"`
	YouTube []string `json:"youtube,omitempty"`
```

Reply:
It's interesting that you can create links to YouTube videos. 
I've added it to the reference acquisition process, so please check the PR.